client/packages/common/src/intl/locales/en/common.json

```diff
@@ -143,6 +144,7 @@
   "heading.user-sync": "User Synchronisation Status",
   "info.server-restarting": "Server restarting...",
   "label.abbreviation": "Abbreviation",
+  "label.programs-immunisations": "Immunisations",
```

Comment:
@adamdewey @mark-prins are you happy calling Immunisation Programs `Immunisations`? Or should we just display `Programs` ? Maybe `Vaccine Programs`?

Reply:
I like the single word options, and `Programs` already has other uses, so I would stick with `Immunizations`. Any thoughts about using US spelling? There are US based funders, but the users..? Which version of English would they typically use? 🤔

Reply:
I agree with `Immunizations` but would like to see it under a `Programs` Nav menu item if possible?

That would leave room to add `HIV`, `TB` as sub-menu items alongside `Immunizations` in the future. 

US English is more common world-wide than British English, especially as a second language, so I would stick with US as default?

Reply:
I.e.

- Programs
  - Immunizations
  - HIV
  - TB

Reply:
I wonder if we should look at consolidating english vs US spelling across the entire app?

I bet there is a tool for that somehow

FYI
https://github.com/msupply-foundation/open-msupply/issues/4046